doc/interop-test-descriptions.md

```diff
@@ -392,22 +392,24 @@ Asserts:
 * clients are free to assert that the response payload body contents are zero
   and comparing the entire response message against a golden response
 
-### Metadata (TODO: fix name)
+### custom_metadata
 
-Status: Not yet implementable
+Status: Ready for implementation, beta requirement.
```

Comment:
For a completed test, the status can be removed. I had just thrown it there for the tests that had major issues.